internal/graphql/resolver/kubernetes.go

```diff
@@ -187,3 +189,58 @@ func (r *Resolver) getScopes(ctx context.Context, name, id *string, trim *bool)
 	}
 	return
 }
+
+func (r *Resolver) getKubectlDescribe(ctx context.Context, name string, typeArg string, namespace string) (*model.KctlDescribeDetails, error) {
+	var ResourceMap = map[string]describe.DescribeType{
+		"Pod":                       describe.Pod,
+		"Deployment":                describe.Deployment,
```

Comment:
Does it mean that you can't write `describe pod xxx` in cli? I mean does it only identifies the capitalised letters?

Reply:
> Does it mean that you can't write `describe pod xxx` in cli? I mean does it only identifies the capitalised letters?

Type of resource needs to be sent to the graphql query, it needs to be capitalised.